Fix style in REST API usage (#2093)
ezsystems/developer-documentation: docs/api/rest_api/rest_api_usage/rest_requests.md

    If applicable, both methods are always mentioned in the specifications.

Unsafe methods require a CSRF token if [session-based authentication](rest_api_authentication.md#session-based-authentication) is used.

### OPTIONS method

> Any URI resource that the REST API responds to also responds to an `OPTIONS` request.

Reviewer: ```suggestion
Any URI resource that the REST API responds to an `OPTIONS` request.
```

Author: @MagdalenaZuba
```suggestion
Any URI resource of the REST API responds to the `OPTIONS` method.
```

Reviewer: ```suggestion
Any URI resource used by the REST API responds to an `OPTIONS` request.
```

Author: ```suggestion
Any REST API URI responds to an `OPTIONS` request.
```
